conf/pipeline/viralrecon/genomes.config

```diff
@@ -90,6 +90,12 @@ params {
             scheme     = 'nCoV-2019'
           }
         }
+        'atoplex' {
+            fasta      = 'https://github.com/nf-core/test-datasets/raw/viralrecon/genome/MN908947.3/primer_schemes/artic/nCoV-2019/V1200/nCoV-2019.reference.fasta'
+            gff        = 'https://github.com/nf-core/test-datasets/raw/viralrecon/genome/NC_045512.2/GCF_009858895.2_ASM985889v3_genomic.200409.gff.gz'
+            primer_bed = 'https://github.com/nf-core/test-datasets/raw/viralrecon/genome/NC_045512.2/amplicon/nCoV-2019.atoplex.V1.bed'
```

Comment:
Can't access this file

Reply:
Appologies, I will fix it ASAP

Reply:
Ping me when you're good to go, and I'll have another look

Reply:
ok, so the issue was that there was a space in the filename, this is now fixed

Reply:
Appologies agai, I'm having a hectic summer :pray: Thank you very much for the fix!!!!